x-pack/plugin/esql/src/main/java/org/elasticsearch/xpack/esql/plugin/EsqlFeatures.java

```diff
@@ -14,13 +14,41 @@
 import java.util.Map;
 
 public class EsqlFeatures implements FeatureSpecification {
```

Comment:
As this class becomes our history tracker, the more info (comments) we have in it, the better.